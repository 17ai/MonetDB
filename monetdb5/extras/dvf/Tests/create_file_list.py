import os, sys, zipfile

dest = os.environ['TSTSRCDIR']
archive = os.path.join(dest, 'example_mseed_files.zip')
if not os.path.exists(archive):
        sys.exit(1)

z = zipfile.ZipFile(archive)
z.extractall(dest)
fl = open(os.path.join(dest, 'example_mseed_file_list.txt'), 'w')
for f in z.namelist():
        fl.write(str(os.path.join(dest, f)) + '\n')
z.close()
