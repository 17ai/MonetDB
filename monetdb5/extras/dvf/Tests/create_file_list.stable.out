stdout of test 'create_file_list` in directory 'extras/dvf` itself:


# 16:20:50 >  
# 16:20:50 >  "/usr/bin/python" "create_file_list.py" "create_file_list"
# 16:20:50 >  


# 16:20:50 >  
# 16:20:50 >  "Done."
# 16:20:50 >  

